openapi.yaml

```diff
@@ -21,6 +21,8 @@ tags:
     description: Team operations
   - name: Environment
     description: Environment operations
+  - name: ACL
```

Comment:
Unsure if this needs to be added as a tag

Reply:
Looks ok to me.